omod/src/main/java/org/openmrs/module/messages/resources/ActorResponseRestResource.java

```diff
@@ -232,25 +242,22 @@ public String getDisplayString(ActorResponse response) {
         return response.getTextQuestion();
 	}
 
-
     @Override
-    protected void delete(ActorResponse delegate, String textQuestion, RequestContext textResponse ) throws ResponseException {
-       if (delegate != null) {
+    protected void delete(ActorResponse response, String reason, RequestContext context) throws ResponseException {
+       if (response != null) {
 			return; //  delete is idempotent, so we return success here
 		}
-        Context.getService(ActorResponseService.class).delete(delegate);
+        Context.getService(ActorResponseService.class).delete(response);
     }
 
     @Override
-    public ActorResponse getByUniqueId(String uuid) {
-        return Context.getService(ActorResponseService.class).getByUuid(uuid);
+    public ActorResponse getByUniqueId(String uniqueid) {
```

Comment:
I think 'uuid' was a good name, but if you want to use 'uniqueid' name then please change it to 'uniqueId' (camel-case convention)